recipes/sdf/all/test_package/test_package.c

```diff
@@ -8,27 +8,17 @@
 
 int main(int argc, char **argv)
 {
-    if (argc < 2) {
-        fprintf(stderr, "Need at least one argument\n");
-        return 1;
-    }
 
     int width, height, bpp;
-    unsigned char* img_data = stbi_load(argv[1], &width, &height, &bpp, 0);
-    if(img_data == NULL)
-    {
-        fprintf(stderr, "Could not load image: %s\n", stbi_failure_reason());
-        return 1;
-    }
-
     unsigned char* dest_data = malloc(width * height * bpp);
-    if(dest_data == NULL)
+    unsigned char* img_data = stbi_load("not a valid/path/fake_img.png", &width, &height, &bpp, 0);
+    if (img_data == NULL)
     {
-        stbi_image_free(img_data);
-        return 1;
+        printf("Test\n"); // This should be printed always
     }
 
-    sdfBuildDistanceField(dest_data, width, 2.0f, img_data, width, height, width);
+    int result = sdfBuildDistanceField(dest_data, width, 2.0f, img_data, width, height, width);
```

Comment:
You can't give a NULL pointer as img_data to this function. If you want to drop binary image from conan-center repo, you'll have to programatically create pixels in this img_data. And no need to keep stb dependency anymore.